app/controllers/audits_controller.rb

```diff
@@ -0,0 +1,113 @@
+class AuditsController < ApplicationController
+  before_action :check_if_organization_admin
+  before_action :set_audit, only: %i(show edit update destroy finalize)
+
+  def index
+    @selected_location = filter_params[:at_location]
+    @audits = current_organization.audits.filter(filter_params)
+    @storage_locations = Audit.storage_locations_audited_for(current_organization).uniq
+  end
+
+  def show
+    @inventory_items = @audit.storage_location.inventory_items
+  end
+
+  def edit
+    redirect_to audits_path unless @audit&.in_progress?
```

Comment:
This has code smell (like -- it feels weird, but I can't put my finger on it). 

I understand the intent here and I'm not sure there's a better way to do it, it just feels odd -- perhaps when we switch to using a proper state-machine this will be feel right.

Reply:
I'm not sure that's necessary -- I think it's OK to leave it as-is (What do you think @mdworken ?)